addons/actions/src/lib/util/getPropertiesList.js

```diff
@@ -0,0 +1,17 @@
+const { hasOwnProperty } = Object.prototype;
+
+export default function getPropertiesList(value) {
+  const keys = Object.getOwnPropertyNames(value);
```

Comment:
This includes non-enumerable props. I don't think they should be displayed. Just `for ... in` should be enough

Reply:
I tried to get the same behavior as in the console for something like the Math object, which does show the non-enumerable properties, otherwise it would just be empty.

Reply:
console shows them in half-transparent style, to indicate that they're somewhat secondary. And I can't imagine someone using Math object as event handler attribute. Let's not add non-enumerable props unless we find a usecase.